### ISSUE TITLE
Temporary target from AAPSClient

### ISSUE THREAD

**Issue by vmichalec** [nightscout/AndroidAPS#3118]
* AAPS Version: 3.2.0.2
* Connection using NSClientV3 over websocket, temp target sync enabled
* NS 15.0.3

Let's assume aaps with SMB disabled with high temp target. We have active temp target of 8 and 10 minutes remaining.
On AAPS side I can just extend this temp target by entering new one without issues below.

On NSClient side I can also set new temp target but there is different behavior:

1. during the sync on AAPS side there can be SMB issued in between those two high temp targets (looks like cancelling current TT, running the loop and maybe give some insulin, setting new TT that is coming).
2. if there is active automation rule to set temp target for example to 5, this one wins and suppress the one coming from remote (such active rule effectively blocks manually set TT from remote).

**Comment by painbrain81** [comment]
it's solved with 3.2.0.3 i think

**Comment by MilosKozak** [comment]
@vmichalec can you re-test with 3.2.0.3 ?

**Comment by vmichalec** [comment]
Yes, I will update tomorrow and check then.

**Comment by vmichalec** [comment]
I tested it shortly on 3.2.0.3:
Issue 1) I didn't noticed this one again.
Issue 2) Still there. I have an automation rule from 21:00 for TT of 5.0 and this rule mostly wins over TT from remote.
I'm sending logs to email.

**Comment by MilosKozak** [comment]
ad 2) i need screenshots what happened (overview, treatments - TT), exact automation rule, then logs with time of occurence

**Comment by vmichalec** [comment]
Ok, fresh case, started after 15:00
- I have created a rule named `TT_DOWN_TO_5` (glucose >= 5.5 triggers TT 5.0 for 30 min)
-  Here is the user activity log on the `AAPS` side
   - 15:03 `TT_DOWN_TO_5` is executed
   - 15:06 I have just set TT on the `AAPS` phone

![Screenshot_20240105-151427](https://github.com/nightscout/AndroidAPS/assets/24793786/9fe8d05b-b2ce-4ee4-ab1d-35a049ea9358)

- And the TT list on the `AAPS` phone

![Screenshot_20240105-151504](https://github.com/nightscout/AndroidAPS/assets/24793786/1f6082ea-1811-4f0b-bbd5-8d120ac834c4)

- On the `AAPSClient` side
   - 15:10:16 I have set TT 

![Screenshot_20240105_151432_AAPSClient](https://github.com/nightscout/AndroidAPS/assets/24793786/f4bf4573-3508-42be-a12c-e7251967cb0e)

- And the TT list on the `AAPSClient` phone

![Screenshot_20240105_151511_AAPSClient](https://github.com/nightscout/AndroidAPS/assets/24793786/14583bbd-d67b-4539-8f34-dcfa54e3c658)

- Finally we can return to the first/second screenshot of `AAPS` and see what happened after 15:10:16
   - 15:10:22 `TT_DOWN_TO_5` is executed
   - on the TT tab we can see two TT starting at 15:10 and this is the issue, simply said - automation rule TT wins over the TT from the `AAPSClient`

I have also noticed few exceptions like this one in the log:
```
15:10:26.615 [RxCachedThreadScheduler-47] D/CORE: [UserEntryLoggerImpl$log$2.accept():61]: FAILED USER ENTRY: java.lang.NullPointerException: Parameter specified as non-null is null: method app.aaps.database.impl.Converters$ValueWithUnitWrapper.<init>, parameter wrapped [UserEntry(id=0, timestamp=1704386157814, utcOffset=3600000, action=TT, source=NSClient, note=, values=[TherapyEventTTReason(value=HYPOGLYCEMIA), Mgdl(value=144.0), null, Minute(value=60)])]
```
To summarize my config:
- aaps & client: 3.2.0.3
- nightscout 15.0.3
- there are two aaps-clients with full rights connected to this nightscout (subjects `vlado-full`, `betuta-full` )
- I just wonder why there are so many logs rows with `"modifiedBy":"vlado-full"` or even `"modifiedBy":"betuta-full"` when I set TT from client phone `vlado-full` only once and client phone `betuta-full` did not changed anything. Please only check if works as it should.
- I'm sending logs to email.

**Comment by MilosKozak** [comment]
put the log here pls. I cannot find it in email

**Comment by vmichalec** [comment]
[Issue_3118.zip](https://github.com/nightscout/AndroidAPS/files/14234003/Issue_3118.zip)
